v0/destinations/mssql/transform.js

```diff
@@ -0,0 +1,23 @@
+const { processWarehouseMessage } = require("../../../warehouse");
+
+const mssql = "mssql";
+
+function processSingleMessage(message, options) {
+  return processWarehouseMessage(message, options);
+}
+
+function getDataTypeOverride(val, options) {}
+
+function process(event) {
+  const whSchemaVersion = event.request.query.whSchemaVersion || "v1";
+  const whStoreEvent = event.destination.Config.storeFullEvent === true;
+  const provider = mssql;
+  return processSingleMessage(event.message, {
+    whSchemaVersion,
+    whStoreEvent,
+    getDataTypeOverride,
+    provider
```

Comment:
@bipin-rudder can you merge latest master into this? This function should also take a new argument to indicate sourceCategory

Reply:
done